assets/css/archive.css

```diff
@@ -3,7 +3,7 @@
 
 .travel-date-input {
-    line-height: 0.125rem;
+    line-height: 1.125rem;
```

Comment:
@kienstra Looks like this CSS condition can be just removed instead of changing since within the CSS it already has `.travel-date-input{line-height:1.125rem;...}`. Seems to be working with removing the condition, too:
![screen shot 2018-05-07 at 10 32 19 pm](https://user-images.githubusercontent.com/3294597/39720937-8c17b7ae-5246-11e8-8311-f13b930c8884.png)

Reply:
Good point, @miina. I'm removing it now.